spec/functional/dsl/reboot_pending_spec.rb

```diff
@@ -79,7 +78,9 @@ def run_ohai
 
     describe "when there is nothing to indicate a reboot is pending" do
       it "should return false" do
-        skip "reboot pending" if @any_flag.any? { |_, v| v == true }
+        skip "reboot pending" if registry_value_exists?('HKLM\SYSTEM\CurrentControlSet\Control\Session Manager', { name: "PendingFileRenameOperations" }) ||
+          registry_key_exists?('HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\WindowsUpdate\Auto Update\RebootRequired') ||
+          registry_key_exists?('HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Component Based Servicing\RebootPending')
```

Comment:
I realize now we weren't testing for this, so the any_flag pattern wouldn't have skipped for it. There's probably a reason we didn't have a test but fair to skip for it.